pkg/autoops/cmd/server/server.go

```diff
@@ -188,14 +189,24 @@ func (s *server) Run(ctx context.Context, metrics metrics.Metrics, logger *zap.L
 	}
 	u.Path = path.Join(u.Path, webhookPath)
 
-	kmsClient, err := kms.NewKeyManagementClient(ctx)
-	if err != nil {
-		return err
+	// TODO: Get cloudService from command-line flags
+	cloudService := "gcp"
```

Comment:
Instead of setting it here, you can use the Default property in the `cmd.Flag` even if there is no flag set in the `deployment.yaml`.

E.g.
```go
cloudService: cmd.Flag("cloud-service", "Cloud Service info").Default("gcp").String()
```